types/max-api/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for max-api 1.1
+// Project: https://docs.cycling74.com/nodeformax/api/index.html
+// Definitions by: Geof Holbrook <https://github.com/geofholbrook>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export const MESSAGE_TYPES: Record<string, string>;
+export const POST_LEVELS: Record<string, string>;
+
+export function addHandler(msg: MessageIdentifier, handler: MessageHandler): void;
+export function addHandlers(handlers: Record<string, MessageHandler>): void;
+export function getDict(id: string): Promise<Dict>;
+export function outlet(...args: any[]): Promise<null>;
```

Comment:
this can be legit, but is that an intent to resolve with null-ish like value in the actual code? (so the type is more like 'void', but bindings enforce some return type)

Reply:
You are correct! In the node for max documentation it is claimed that these return null, but experimentally they return void. I fixed those return types